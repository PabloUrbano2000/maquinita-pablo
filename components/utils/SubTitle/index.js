import { styled } from '@mui/material/styles';

const SubTitle = styled('span')({
  color: 'black',
  fontWeight: 'bold',
  fontFamily: 'Gibson'
});

export default SubTitle;
