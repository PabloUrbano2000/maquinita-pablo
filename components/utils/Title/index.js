import { styled } from '@mui/material/styles';

const Title = styled('span')({
  color: 'black',
  fontWeight: 'bold',
  fontFamily: 'Gibson-BoldItalic'
});

export default Title;
