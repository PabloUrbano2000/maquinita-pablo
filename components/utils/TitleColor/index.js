import { styled } from '@mui/material/styles';

const TitleColor = styled('span')({
  color: '#d71919',
  fontWeight: 'bold'
  // fontFamily: 'Gibson'
});

export default TitleColor;
