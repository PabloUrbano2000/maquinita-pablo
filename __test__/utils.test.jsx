import { render } from '@testing-library/react';
import Title from '../components/utils/Title';
import SubTitle from '../components/utils/SubTitle';
import TitleColor from '../components/utils/TitleColor';

it('render component utils/Title', () => {
  const { container } = render(<Title />);
  expect(container).toMatchSnapshot();
});

it('render component utils/SubTitle', () => {
  const { container } = render(<SubTitle />);
  expect(container).toMatchSnapshot();
});

it('render component utils/TitleColor', () => {
  const { container } = render(<TitleColor />);
  expect(container).toMatchSnapshot();
});
