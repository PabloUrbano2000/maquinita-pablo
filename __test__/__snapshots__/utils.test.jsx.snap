// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render component utils/SubTitle 1`] = `
<div>
  <span
    class="css-8rchdx"
  />
</div>
`;

exports[`render component utils/Title 1`] = `
<div>
  <span
    class="css-yzbrn7"
  />
</div>
`;

exports[`render component utils/TitleColor 1`] = `
<div>
  <span
    class="css-1sah1m0"
  />
</div>
`;
